### PR TITLE
Release bigquery 1.8.0

### PR DESCRIPTION
**Pull request by crwilcox**
This pull request was generated using releasetool.

### CODE REVIEW

File: bigquery/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,30 @@
...
+## 1.8.0
+
+12-10-2018 12:39 PST
+
+
+### Implementation Changes
+- Add option to use BQ Storage API with to_dataframe ([#6854](https://github.com/googleapis/google-cloud-python/pull/6854))
+- Fix exception type in comment ([#6847](https://github.com/googleapis/google-cloud-python/pull/6847))
+- Add to_bqstorage to convert from Table[Reference] google-cloud-bigquery-storage reference ([#6840](https://github.com/googleapis/google-cloud-python/pull/6840))
+- Use moved iam.policy now at google.api_core.iam.policy ([#6741](https://github.com/googleapis/google-cloud-python/pull/6741))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]:
```suggestion
- Import `iam.policy` from `google.api_core`. ([#6741](https://github.com/googleapis/google-cloud-python/pull/6741))
```

File: bigquery/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,30 @@
 
 [1]: https://pypi.org/project/google-cloud-bigquery/#history
 
+## 1.8.0
+
+12-10-2018 12:39 PST
+
+
+### Implementation Changes
+- Add option to use BQ Storage API with to_dataframe ([#6854](https://github.com/googleapis/google-cloud-python/pull/6854))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]: Use backticks to avoid breaking Markdown.

File: bigquery/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,30 @@
...
+- Fix exception type in comment ([#6847](https://github.com/googleapis/google-cloud-python/pull/6847))
+- Add to_bqstorage to convert from Table[Reference] google-cloud-bigquery-storage reference ([#6840](https://github.com/googleapis/google-cloud-python/pull/6840))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]: Backticks.

File: bigquery/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,30 @@
...
+- Use moved iam.policy now at google.api_core.iam.policy ([#6741](https://github.com/googleapis/google-cloud-python/pull/6741))
+- Add avro logical type control for load jobs. ([#6827](https://github.com/googleapis/google-cloud-python/pull/6827))
+- Allow setting partition expiration to 'None'. ([#6823](https://github.com/googleapis/google-cloud-python/pull/6823))
+- Add 'retry' argument to '_AsyncJob.result'. ([#6302](https://github.com/googleapis/google-cloud-python/pull/6302))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]: Backticks.

**Comment by crwilcox** [comment]
Release is at https://github.com/googleapis/google-cloud-python/releases/tag/bigquery-1.7.1

**Pull request merged**